frontend/src/Components/loginForm.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+
+
+export const LoginForm = () => {
+    const [username, setUsername] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+          axios.post("http://localhost:8000/login/password", {
+            username,
+            password,
+          })
+          .then(response => {
+            console.log(response.data);
+          })
+          .catch(error => {
+            console.error(error);
+          });
+      };  
+    
+    
+    return (
+        <form action="/login/password" method="post" onSubmit={handleSubmit}>
+            <div>
+                <label htmlFor="username">Username</label>
+                <input id="username" name="username" type="text" autoComplete="username" required value={username} onChange={event => setUsername(event.target.value)} />
```

Comment:
i know the onChange here is quite small but it's good practice to try and make the return of the component as structural-only as possible and pull any functions and business logic into separate variables above. so let's pull this onChange out and define it separately!